components/eam/src/physics/crm/crm_physics.F90

```diff
@@ -75,6 +75,8 @@ subroutine crm_physics_register()
    use gator_mod, only: gator_init
 #elif defined(MMF_SAM)
    use setparm_mod      ,   only: setparm
+#elif defined(MMF_SAMOMP)
+   use setparm_mod     ,    only: setparm
```

Comment:
Would it be better to just use:
`#elif defined(MMF_SAM) || defined(MMF_SAMOMP)`
to avoid the duplicate lines?

Reply:
that is great, thanks.